rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -356,21 +356,30 @@ public JobStateData listJobs(String sessionId, String jobId) throws RestExceptio
     }
 
     @Override
-    public String getLiveLogJob(String sessionId, String jobId) throws NotConnectedRestException,
+    public String getLiveLogJob(String sessionId, String jobId, boolean allLogs) throws NotConnectedRestException,
             UnknownJobRestException, PermissionRestException, LogForwardingRestException, IOException {
         try {
             Scheduler scheduler = checkAccess(sessionId, "/scheduler/jobs/" + jobId + "/livelog");
             Session session = sessionStore.get(sessionId);
 
             JobState jobState = scheduler.getJobState(jobId);
             boolean isFinished = jobState != null && jobState.isFinished();
-            int availableLinesCount = session.getJobsOutputController().availableLinesCount(jobId);
-
-            if (!isFinished || availableLinesCount > 0) {
-                return session.getJobsOutputController().getNewLogs(jobId);
+            if (allLogs) {
+                if (!isFinished) {
```

Comment:
why `|| availableLinesCount > 0` could not be applied here?

Reply:
Ok, I will investigate